.gitignore

```diff
@@ -10,6 +10,7 @@ dist/
 
 # Ignore virtual environment
 .venv/
+venv/
```

Comment:
This is my mistake. In [CONTRIBUTING.md](https://github.com/ss77995ss/baseball-stats-python/blob/main/CONTRIBUTING.md) `venv` should be `.venv`

```diff
- python -m venv venv
+ python -m venv .venv
```